handlers.go

```diff
@@ -495,3 +515,76 @@ func (a *autographer) handleGetAuthKeyIDs(w http.ResponseWriter, r *http.Request
 	w.WriteHeader(http.StatusOK)
 	w.Write(signerIDsJSON)
 }
+
+func (a *autographer) handleGetX5U(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		httpError(w, r, http.StatusMethodNotAllowed, "%s method not allowed; endpoint accepts GET only", r.Method)
+		return
+	}
+	if r.Body != nil {
```

Comment:
All of this code can be deleted. There's no `Body` ever sent on `GET`s and we shouldn't do anything with them, anyway. Go will handle the reading of bogus requests when we return the 405.

Reply:
IIUC, technically this is incorrect. The HTTP protocol doesn't prohibit a `GET` from providing a body, it is just that by convention no one does so.

Reply:
Sure, but I forgot to mention that the Go http library will close out the Body for us, anyhow, so it's still redundant.